app-admin/conky/files/conky-1.10.8-virtual-sinks.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/src/pulseaudio.cc b/src/pulseaudio.cc
+index 93226c1b..39546bd1 100644
+--- a/src/pulseaudio.cc
++++ b/src/pulseaudio.cc
+@@ -53,8 +53,10 @@ void pa_sink_info_callback(pa_context *c, const pa_sink_info *i, int eol, void *
+         pdr->sink_mute = i->mute;
+         pdr->sink_card = i->card;
+         pdr->sink_index = i->index;
+-        pdr->sink_active_port_name.assign(i->active_port->name);
+-        pdr->sink_active_port_description.assign(i->active_port->description);
++		if (i->active_port != nullptr) {
++			pdr->sink_active_port_name.assign(i->active_port->name);
++			pdr->sink_active_port_description.assign(i->active_port->description);
++		}
```

Comment:
No expert on this but what about the else clause of the original commit brndnmtthws/conky@2a7226d where sink_active_port_name and sink_active_port_description are erased?

Reply:
Good point, this might leave outdated information in the buffer. I'll add it too.